File: python/programmers/level 0/개미 군단.py
def solution(hp):
    장군 = 5
    병정 = 3
    
    answer = 0
    while True:
        if hp >= 장군:
            answer += hp // 장군
            hp -= 장군 * (hp // 장군)
        elif hp >= 병정:
            answer += hp // 병정
            hp -= 병정 * (hp // 병정)
        else:
            break
    answer += hp
    return answer

File: java/src/main/java/ps_traning/programmers/level_0/개미_군단.java
package ps_traning.programmers.level_0;

public class 개미_군단 {
    public int solution(int hp) {
        int answer = 0;
        answer += (int) hp / 5;
        hp = hp % 5;
        answer += (int) hp / 3;
        hp = hp % 3;
        answer += hp;
        return answer;
    }
}
